libs/sdk-ui-charts/src/highcharts/chartTypes/_chartOptions/chartAxes.ts

```diff
@@ -63,6 +73,22 @@ export function getXAxes(
         }
     }
 
+    if (enableJoinedAttributeAxisName && (isBarChart(type) || isColumnChart(type) || isBulletChart(type))) {
```

Comment:
The `(isBarChart(type) || isColumnChart(type) || isBulletChart(type))` part is repeated on several places in this PR. Can we please create a function for this? So that when a new chart starts supporting this feature we can change as few places as possible?

Reply:
Now there is a function `isSupportingJoinedAttributeAxisName(type)`, which handles the check of the types.